internal/controller/phases.go

```diff
@@ -136,16 +132,36 @@ func (p *phaseReconciler) load(ctx context.Context) (reconcile.Result, error) {
 
 	spec := p.provider.GetSpec()
 
-	// If a configmap selector was specified, use it to find the configmap with provider configuration. This is
-	// a case for "air-gapped" environments. If no selector was specified, use GitHub/Gitlab repository.
-	if spec.FetchConfig != nil && spec.FetchConfig.Selector != nil {
-		log.V(5).Info("Custom ConfigMap was provided for fetching manifests")
+	// Store some provider specific inputs for passing it to clusterctl library
+	p.options = repository.ComponentsOptions{
+		TargetNamespace:     p.provider.GetNamespace(),
+		SkipTemplateProcess: false,
+		Version:             spec.Version,
+	}
 
-		p.repo, err = p.configmapRepository(ctx)
-	} else {
-		p.repo, err = repositoryFactory(p.providerConfig, p.configClient.Variables())
+	return reconcile.Result{}, nil
+}
+
+// load provider specific configuration into phaseReconciler object.
+func (p *phaseReconciler) load(ctx context.Context) (reconcile.Result, error) {
+	log := ctrl.LoggerFrom(ctx)
+
+	log.Info("Loading provider")
```

Comment:
```suggestion
	log.V(5).Info("Loading provider")
```

Reply:
We actually moved these logs to the 0 level some time ago so the operator is not too silent by default

Reply:
@alexander-demicev you mean in all cases? Because I spotted https://github.com/kubernetes-sigs/cluster-api-operator/blob/f51695b89bd11389418d9a8d45f53f88be133f70/internal/controller/phases.go#L142 where it is using the log level, but if that is just a left out then you can disregard my comment

Reply:
I think setting log level is not strictly related to this PR, so for now we can keep it as-is and address it later.